tests/plugins/test_resolve_composes.py

```diff
@@ -1363,7 +1364,10 @@ def run_plugin_with_args(self, workflow, plugin_args=None,
         if expect_error:
             with pytest.raises(PluginFailedException) as exc_info:
                 runner.run()
-            assert expect_error in str(exc_info.value)
+            if hasattr(expect_error, 'search'):  # py2/3 compat, different re.Compile objects
```

Comment:
How about use `six.PY2` or `six.PY3`? No comment is required and the code is much straightforward.

Reply:
Can you provide exaple? I don't think that using six makes code more straightforward in this case

Reply:
Actually, I'd like to see the comment explain *which* Python re.Compile object has  `search` (so I don't have to look it up :p)